.azure-pipelines/client.test.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  name: ''
+  vmImage: ''
+
+jobs:
+  - job: ${{ format('Test_{0}', parameters.name) }}
+    dependsOn:
+      - 'Publish'
+
+    pool:
+      vmImage: ${{ parameters.vmImage }}
+
+    variables:
+      build.path: 'test/services/keyVault'
+
+    strategy:
+      matrix:
+        Node8:
```

Comment:
Supposedly we still support Node6 in the management plane. You may consider adding that to the matrix.

Reply:
I'm not sure if Node6 is in the support matrix we're planning for the future, but KeyVault tests pass in Node6 so it's good to keep running them.  Added Node6 to this PR.